etc/ERC20/package.json

```diff
@@ -6,9 +6,9 @@
   "devDependencies": {
     "@matterlabs/hardhat-zksync-deploy": "^0.6.1",
     "@matterlabs/hardhat-zksync-solc": "0.4.2",
-    "hardhat": "=2.16.0"
+    "hardhat": "=2.18.2"
```

Comment:
![image](https://github.com/matter-labs/zksync-era/assets/12111581/1d0ba2a6-9803-4f12-ba70-8d28101f00a2)

It seems that we have hardhat defined in multiple places (and we may call it from different places via `zk`), and also it's a bit sus that `yarn.lock` wasn't affected.

Reply:
Yup, just bumped it everywhere else too just in case

Reply:
I mean, we probably should bump it in the contracts submodule as well, I guess.

Reply:
Absolutely, I am just trying to see if this change alone will fix the flakiness as the init errors are consistently coming from `etc/ERC20`